src/test/runTest.ts

```diff
@@ -2,7 +2,7 @@ import { spawnSync } from 'child_process';
 import * as path from 'path';
 import * as fs from 'fs';
 
-import { runTests } from 'vscode-test';
+import { runTests } from '@vscode/test-electron';
```

Comment:
Just a question, is this related to the Yarn2 upgrade?

Reply:
The package was apparently migrated it to a different namespace, and the old repository is now deprecated. Same happened with `vsce` => `@vscode/vsce`. Yarn gave warnings about the old ones.

Reply:
(This was a general issue, not a Yarn 2-specific one)